sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2119,24 +2146,30 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     withTable("t", "t1") {
```

Comment:
This test seems checking local paths are actually stored as fully qualified URI. So, here, I used local paths.